pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,75 @@
+# JSON Schema documentation contributing guide 
+
+This guide describes the workflow to contribute to the JSON Schema technical documentation. If you would like to contribute to other aspects of the JSON Schema website, like writing blog articles or publishing a case study, please read the website's [Contributing Guide](/website/CONTRIBUTING.md) for more details. 
```

Comment:
We need to point here or somewhere else (maybe in before you start section) to the Docs style guide here to make sure contributors follow the style guide.